src/background/analysis/analyze.js

```diff
@@ -144,6 +145,15 @@ function resolveBuffer(id, data) {
 async function analyze(request, userData) {
 
   const allEvidence = getAllEvidenceForRequest(request, userData);
+
```

Comment:
Another thought here is that we will be analyzing the same cookies over and over the way you have it set up. I think we should maintain a map of reqUrls -> timestamps for a given browsing session. Then we should only run the cookie analysis once every minute (or even 20 seconds or something). It's just clear that we will run the same analysis for the main first-party cookies many times in a row on page load.